ros/src/.config/rviz/default.rviz

```diff
@@ -6,20 +6,13 @@ Panels:
       Expanded:
         - /Global Options1
```

Comment:
This file is overwriting the default config for rviz which leads to disruptions of the rosbag demo, can you remove it from the PR?

Reply:
sorry for that. I reverted it.